app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ForgotPasswordsController < ApplicationController
+  def create
+    user = User.where(email_address: params[:email_address]).first
```

Comment:
I'm kicking myself because I'm pretty sure I know this and should have followed my instincts. Using find_by() is a better fit because it will return one object. Using where() returns an array, which is why .first is added to. Better to use find_by() when querying for a single object.